CHNGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
```

Comment:
I thought we decided to use the Releases by GitHub?

Reply:
removed